test/callback.cc

```diff
@@ -3,8 +3,8 @@
 #include "gtest/gtest.h"
 #include "stout/borrowed_ptr.h"
 
-using eventuals::Callback;
-
+namespace eventuals {
```

Comment:
As discussed offline, let's put all of these in the `*::test` namespace, e.g., `eventuals::test`.